modules/c/pages/vector-search.adoc

```diff
@@ -1,12 +1,12 @@
 = Vector Search
-:page-status: Beta
+:page-status:
```

Comment:
I suspect this should be commented out rather than left blank, as (weirdly), blank is a "truthy" value.
But I saw you had some back and forth on this earlier, so not important while the release needs to go ahead, can look at it together in a tidy-up phase later 👍